FEM/rf_mfp_new.cpp

```diff
@@ -1053,7 +1053,19 @@ void CFluidProperties::CalPrimaryVariable(
         // MX  m_pcs = PCSGet("HEAT_TRANSPORT");
         m_pcs = PCSGet(pcs_name_vector[i], true);
         if (!m_pcs)
-            return;  // MX
+        {
+            if (pcs_name_vector[i].find("TEMPERATURE1") != std::string::npos)
```

Comment:
why do you compare a process name with TEMPERATURE1? or is just the variable name confusing me?

Reply:
@norihiro-w This name is misleading. I just added another commit to change this name. Thanks for your comment.

Reply:
thank you for the change!